src/main/java/net/galgus/flink/streaming/connectors/http/HTTPSink.java

```diff
@@ -38,6 +38,11 @@ public void invoke(IN value, Context context) throws Exception {
             writer.close();
 
             int status = conn.getResponseCode();
+            log.info("Status={}", status);
+            if(status == 200) {
+                log.info("Message: The request has succeeded" );
```

Comment:
you can merge this line with the line at 41 and just have some log messages that says, what the response code was, which url, and how long did the request take.

Reply:
there is no method to check the request duration so I just calculated the time in the code. The else statement created by the original author already logs the code, url and body so decided not to merge with line 41